cmd/oras/internal/argument/file_unix.go

```diff
@@ -15,12 +15,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package main
+package argument
```

Comment:
package name `argument` seems weird. How about package `file` with func `ParseReference`?

Reply:
You are right. It's related to file reference, now used in argument parsing, can be applied to others like flag parsing if we do have a file reference flag.